SignalServiceKit/src/Messages/OWSMessageSender.m

```diff
@@ -242,8 +242,9 @@ - (void)setBackgroundTaskIdentifier:(UIBackgroundTaskIdentifier)backgroundTaskId
     AssertIsOnMainThread();
 
     // Should only be sent once per operation
-    OWSAssert(_backgroundTaskIdentifier == UIBackgroundTaskInvalid);
-    OWSAssert(backgroundTaskIdentifier != UIBackgroundTaskInvalid);
+    // FIXME SHARINGEXTENSION
+//    OWSAssert(_backgroundTaskIdentifier == UIBackgroundTaskInvalid);
+//    OWSAssert(backgroundTaskIdentifier != UIBackgroundTaskInvalid);
```

Comment:
`OWSAssert(!CurrentAppContext() || task identifiers are valid)`

Reply:
I guess it is that simple, huh.

I'm assuming you meant:
```
OWSAssert(!CurrentAppContext().isMainApp || _backgroundTaskIdentifier == UIBackgroundTaskInvalid);
```